### PR TITLE
Include core-js package in package.json

### LINKED ISSUE #332: simplebar-vue does not work with core-js@3
**Describe the bug**
I'm updating my app to `core-js@3` and `simplebar-vue` does not work with it. Building with webpack produces error: 

```
ERROR in ../node_modules/simplebar-vue/dist/simplebar-vue.esm.js
Module not found: Error: Can't resolve 'core-js/modules/es7.promise.finally' in '....\node_modules\simplebar-vue\dist'
```

**To Reproduce**
1. Install and import simplebar-vue
2. Use core-js@3 in your app
3. Try building

**Expected behavior**
No error should be produced 🙂 

**Additional context**
Looks like simplebar-vue does not specify core-js as a dependency. And .babelrc does not set target core-js version too.
Checking out project and trying to built simplebar-vue I get a warning:

```
WARNING: We noticed you're using the `useBuiltIns` option without declaring a core-js version. Currently, we assume version 2.x when no version is passed. Since this default version will likely change in future versions of Babel, we recommend explicitly setting the core-js version you are using via the `corejs` option.

You should also be sure that the version you pass to the `corejs` option matches the version specified in your `package.json`'s `dependencies` section. If it doesn't, you need to run one of the following commands:

  npm install --save core-js@2    npm install --save core-js@3
  yarn add core-js@2              yarn add core-js@3
```

So looks like it is needed to specify core-js version in package.json and .babelrc for simplebar-vue.

**Your environment**

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar        | simplebar-vue@1.0.0
| Browser          | n/a
| npm/Yarn         | yarn@1.16.0
| Operating System | Windows 10

### PR DESCRIPTION
**Pull request by Demivan**
Fixes #332

### CODE REVIEW

**Comment by Grsmto** [comment]
Hi,
Thanks for your contribution.
Does this actually fixes the issue? As I suppose you'll have both core-js 2 and 3 as dependency but they can't work concurrently so I'm not sure to see how that would fix it?
If you check the `simplebar` package, there is the corejs 3 configuration in place there. Maybe there should just be the same thing in the vue package?

**Comment by Demivan** [comment]
Yeah. It fixed this issue for me.
Right now issue is that `simplebar-vue` does not have `core-js` package specified as dependency at all. It just uses hoisted package version, so if app has `core-js@3` in its `package.json` it will try to use that. It does not use `core-js@3` dependency from `simplebar` package. And even if it did it would not work because current babel config for `simplebar-vue` generates `core-js` imports using version 2 format.
Best solution would, probably, be to update vue related dependencies (babel preset) for `simplebar-vue` and switch to `core-js@3`.
I can update this pull-request to include that.

**Comment by Demivan** [comment]
Just checked vue packages and `babel-preset-app@3.8.0` is not updated to `core@3` yet, version 4 where it is updated is in alpha.

**Comment by Demivan** [comment]
I have removed `@vue/cli-service` and related packages - they were used mostly for tests. Instead I have added `vue-jest` and used `jest` directly for tests. Additionally I have removed `@vue/babel-preset-app` and used just `@babel/preset-env`. This allowed me to switch to `core-js@3`.
Tests are passing, but I will test tomorrow on my project for both `core-js@2` and `core-js@3`
Another benefit of these changes are 3.5k lines removed from yarn.lock :slightly_smiling_face:

**Comment by Demivan** [comment]
Just tested and it worked in both cases: when app is targeting core-js@2 and when core-js@3

File: packages/simplebar-vue/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -1,49 +1,110 @@
...
 
 exports[`simplebar renders with options 1`] = `
-<div data-simplebar="" data-simplebar-auto-hide="false">
-  <div class="simplebar-wrapper">
-    <div class="simplebar-height-auto-observer-wrapper">
-      <div class="simplebar-height-auto-observer"></div>
+<div
+  data-simplebar=""
+  data-simplebar-auto-hide="false"
+>
```
Comment (Grsmto) [inline packages/simplebar-vue/__snapshots__/index.test.js.snap]: hey, do you know why these changes happened? It seems like a prettifier passed here.
In theory we have a precommit hook with Prettier so idk what happened. if you could revert this before merge.

Comment (Demivan) [reply packages/simplebar-vue/__snapshots__/index.test.js.snap]: It was because I used `wrapper.element` when making snapshots.
https://vue-test-utils.vuejs.org/guides/testing-single-file-components-with-jest.html#snapshot-testing
I added `jest-serializer-vue` now so snapshots are the same as before.

Comment (Grsmto) [reply packages/simplebar-vue/__snapshots__/index.test.js.snap]: Hey actually I thought this was the source, I didn't realise I was commenting on a snapshot file!
That's totally fine to have these changes then haha
